diff -Nur -x '*.orig' -x '*.rej' e-smith-domains-1.4.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/domains mezzanine_patched_e-smith-domains-1.4.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/domains
--- e-smith-domains-1.4.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/domains	2008-03-12 09:25:36.000000000 -0600
+++ mezzanine_patched_e-smith-domains-1.4.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/domains	2008-03-12 09:25:14.000000000 -0600
@@ -88,10 +88,6 @@
         </trans>
     </entry>
     <entry>
-        <base>CREATE</base>
-        <trans>Add</trans>
-    </entry>
-    <entry>
         <base>DOMAIN_NAME_VALIDATION_ERROR</base>
         <trans>
             Error: unexpected or missing characters in domain name
diff -Nur -x '*.orig' -x '*.rej' e-smith-domains-1.4.0/root/etc/e-smith/web/functions/domains mezzanine_patched_e-smith-domains-1.4.0/root/etc/e-smith/web/functions/domains
--- e-smith-domains-1.4.0/root/etc/e-smith/web/functions/domains	2005-08-25 11:51:33.000000000 -0600
+++ mezzanine_patched_e-smith-domains-1.4.0/root/etc/e-smith/web/functions/domains	2008-03-12 09:25:11.000000000 -0600
@@ -114,7 +114,7 @@
             <description>DESC_NAMESERVERS</description>
         </field>
 
-        <subroutine src="print_button('CREATE')" />
+        <subroutine src="print_button('ADD')" />
     </page>
 
     <page name="DOMAINS_PAGE_MODIFY"
